index.html

```diff
@@ -60,7 +61,7 @@ <h3>Easy to HTML</h3>
         <div class='website-navigation'>
           <div class='row buttons-container'>
             <div class='col-md-6 col-lg-6 col-sm-12 col-xs-12 mt-3'>
-              <a href='puzzles/puzzle1.html'><button class='btn btn-default btn-block'>Puzzle One Prototype</button></a>
+              <a href='campers/campers.html'><button class='btn btn-default btn-block'>Campers One Prototype</button></a>
```

Comment:
likewise, I think puzzles/puzzle1.html or challenges/challenge1.html is more accurate because it implies a later puzzle2 or challenge2